packages/apps/plugins/plugin-space/src/translations.ts

```diff
@@ -45,6 +45,8 @@ export default [
         'duplicate object label': 'Duplicate item',
         'delete collection label': 'Delete collection',
         'delete object label': 'Delete item',
+        'collection deleted label': 'Collection deleted',
+        'object deleted label': 'Item deleted',
```

Comment:
realizing that we use `item` in most user-facing language for objects...except `unnamed object label` 🤔

Reply:
🤔